packages/http/src/forwarder/index.ts

```diff
@@ -1,54 +1,79 @@
 import { IPrismComponents } from '@stoplight/prism-core';
 import { IHttpOperation } from '@stoplight/types';
-import fetch from 'node-fetch';
+import fetch, { Response } from 'node-fetch';
 import * as Either from 'fp-ts/lib/Either';
 import * as TaskEither from 'fp-ts/lib/TaskEither';
-import { defaults, omit } from 'lodash';
+import * as ReaderTaskEither from 'fp-ts/lib/ReaderTaskEither';
+import { defaults, omit, pick } from 'lodash';
 import { format, parse } from 'url';
 import { IHttpConfig, IHttpRequest, IHttpResponse } from '../types';
 import { posix } from 'path';
 import { parseResponse } from '../utils/parseResponse';
 import { pipe } from 'fp-ts/lib/pipeable';
+import { logRequest, logResponse } from '../utils/logger';
+import { Logger } from 'pino';
+import chalk from 'chalk';
```

Comment:
While we can debate whether it shall or not, I added chalk dependency do `cli` and `http`.

I aggree that this might be not the best idea to have `chalk` included in `http`. `http` is a library and shall not output anything to console by design. On the other hand, I don't have a better solution to that?